crates/diesel_utils/replaceable_schema/triggers.sql

```diff
@@ -696,3 +696,126 @@ CREATE TRIGGER search_combined_community_score
     AFTER UPDATE OF users_active_month ON community
     FOR EACH ROW
     EXECUTE FUNCTION r.search_combined_community_score_update ();
+-- Increment / decrement multi_community counts
```

Comment:
This is verbose, but its how we should be doing all triggers. Simple +1, -1 functions, and atomic triggers that only fire on very minimal conditions. Its verbose but performant.